### PR TITLE
Merge v0.7rc stuff to master

### PR DESCRIPTION
**Pull request by deepanshs**
Port changes that will go into the v0.7 release.

- [x] Update "Changelog" file.
- [x] Check documentation.

### CODE REVIEW

File: docs/user_guide/spin_system_distributions/single_site_sys_gen.rst [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+      - Type
+      - Description
+
+    * - ``isotope``
+      - ``str`` or list of ``str``
+      - A **required** string or list of strings representing the label of the *isotope* attribute
+        of the :ref:`site_api` (e.g. ``"1H"`` or ``["29Si", "17O"]``).
+
+    * - ``isotropic_chemical_shift``
+      - ``float``, list of ``float`` or numpy array
```
Comment (mgiammar) [inline docs/user_guide/spin_system_distributions/single_site_sys_gen.rst:122]: add comma

File: docs/user_guide/spin_system_distributions/single_site_sys_gen.rst [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+        or lists/numpy arrays of floats. The default is ``None``.
+
+    * - ``quadrupolar``
+      - ``dict``
+      - An *optional* dictionary representing the *quadrupolar* attribute of the
+        :ref:`site_api` where the keys are valid :ref:`sy_api` attributes and the values are floats
+        or lists/numpy arrays of floats. The default is ``None``.
+
+    * - ``abundance``
+      - ``float``, list of ``float`` or numpy array
```
Comment (mgiammar) [inline docs/user_guide/spin_system_distributions/single_site_sys_gen.rst:146]: Add comma

File: docs/credits/license.rst [hunk 3]
```diff
@@ -6,7 +6,7 @@ Mrsimulator License
 
 Mrsimulator is licensed under BSD 3-Clause License
 
-Copyright (c) 2019-2021, Mrsimulator Developers,
+Copyright (c) 2019-2022, Mrsimulator Developers,
```
[the remark this answers is not in the document]
Comment (mgiammar) [reply docs/credits/license.rst:9]: Used ``|today|``

File: docs/user_guide/method/method.rst [hunk 4]
```diff
@@ -0,0 +1,79 @@
+.. _method_documentation:
```
[the remark this answers is not in the document]
Comment (deepanshs) [reply docs/user_guide/method/method.rst:1]: @mgiammar Let's continue with this documentation in a new PR.

**Review by mgiammar — APPROVED** [review]

**Pull request merged**